submissions/BukkyOyetimehin/database/init.sql

```diff
@@ -0,0 +1,173 @@
+-- Creating the Databse
+CREATE DATABASE fashi_online_store;
+SHOW DATABASES;
+
+-- Creating the Tables
+
+USE fashi_online_store;
+-- Categories Table
+CREATE TABLE categories (
+	id INT NOT NULL AUTO_INCREMENT,
+	category_name VARCHAR(100) NOT NULL,
+    created DATETIME,
+	PRIMARY KEY (id)	
+);
+ALTER TABLE categories CHANGE created  created_at DATETIME;
+SHOW COLUMNS FROM categories;
+
+INSERT INTO categories (category_name, created_at)
+VALUES ('Men Apparel', now()),
+('Women Apparel', now()),
+('Men Accessories', now()),
+('Women Accessories', now()),
+('Sports & Outdoors', now());
+
+SELECT * FROM categories;
+
+-- AdminLogin Table
+CREATE TABLE admin_Login (
+	id INT NOT NULL AUTO_INCREMENT,
+	full_name VARCHAR(40) NOT NULL,
+    admin_email VARCHAR (30) NOT NULL,
+    admin_password VARCHAR(20) NOT NULL,
+    admin_phone VARCHAR(30) NOT NULL,  
+    created_at DATETIME,
+	PRIMARY KEY (id)
+    );
+USE fashi_online_store;
+ALTER TABLE admin_Login
+RENAME TO admin_login;
+SHOW TABLES;
+
+INSERT INTO admin_login (full_name, admin_email, admin_password, admin_phone, created_at)
+VALUES ('Ben Ellison', 'bellisongmail.com', 'info@1234', '950-830-3729', now()),
+('Danika Marshall', 'dmarshall@gmail.com', 'workflow%#1105', '226-813-4145', now()),
+('Linda McGrath', 'lmcgrath@gmail.com', 'shalom**203', '430-646-2808', now());
+
+SELECT * FROM admin_login;
+
+-- Products Table
+CREATE TABLE products (
+	id INT NOT NULL AUTO_INCREMENT,
+	product_name VARCHAR(100) NOT NULL,
+    Brand VARCHAR(100), 
+    category_id INT NOT NULL,
+	product_description VARCHAR(250) NOT NULL,
+	product_image BLOB NOT NULL,
+	product_price DECIMAL(10,2) NOT NULL,    
+    stock_level INT NOT NULL,
+    product_status VARCHAR(40) NOT NULL,
+    admin_login_id INT NOT NULL,
+    created_at DATETIME,
+    PRIMARY KEY(id),
+    FOREIGN KEY (category_id) REFERENCES categories (id),
+	FOREIGN KEY (admin_login_id) REFERENCES admin_login (id)
+);
+USE fashi_online_store;
+ALTER TABLE products CHANGE Brand brand VARCHAR(100);
```

Comment:
`brand` should not be a column on the `products` table. Or, is there a place on the task where is it mentioned